local.c

```diff
@@ -1575,6 +1618,14 @@ static int detect_and_move_global_attrs(struct iio_device *dev)
 	return 0;
 }
 
+static int cmp_iio_buffer_attr(const void *p1, const void *p2)
```

Comment:
Would be good to re-use the same function for all which just do strcmp().

Reply:
I wasn't sure if I should do that - in case the structures changed later...

-Robin